README.md

```diff
@@ -475,6 +475,18 @@ end
 - When erlang version 17.3 is used, an error message `(MatchError) no match of right hand side value: ""` can be shown. Refer to issue #14 for the details.
     - https://github.com/parroty/excoveralls/issues/14
 
+### Silence OTP Cover Warnings
+To remove OTP warnings about modules or specific logging, please open a GitHub issue in the Erlang OTP repository. Alternatively you can copy the `cover.erl` file under `src/` of your Elixir project and modify it to remove the warnings.
```

Comment:
I think the `open a GitHub issue in the Erlang OTP repository` is a separate discussion from the procedure itself. Can we exclude it this part? (may be like the following?)

> To remove OTP warnings about modules or specific logging, you can copy the `cover.erl` file under `src/` of your Elixir project and modify it to remove the warnings, as a tentative solution.

Reply:
sure, it might be easier for you to cherry pick what's needed and update README directly.

Reply:
OK. I'll update the README separately (thank you for the contribution).
I tried to update the PR itself, but couldn't do that, maybe because it's coming from fork.